Assets/Scripts/Actors/ActorComponents/PlayerActorResources.cs

```diff
@@ -174,7 +174,7 @@ void OnTriggerEnter( Collider other )
 			PickupResource( resourceComponent.resourceData );
 
 			Destroy( other.gameObject );
-			WadeUtils.TempInstantiate( resourcePopPrefab, other.transform.position, Quaternion.identity, 1f );
+			SoundManager.instance.Play3DSoundAtPosition( resourcePopAudio, other.transform.position );
```

Comment:
@brickman1444 How do we want to do singletons in the project? I'd argue that we have public static members that automatically act on the instance, rather than having to manually retrieve the instance every time you use the singleton. Chris and I discussed this at GGJ, but I don't think we've brought it up for this project.